### DISCUSSION TITLE
Search broken on website

### DISCUSSION THREAD

**Discussion by ps-20x** [serversideup/spin#67] [General]
On the [website](https://serversideup.net/open-source/spin/docs) I think the search is broken

![image](https://github.com/serversideup/spin/assets/126659880/99d046cd-b0bb-4b44-b177-774e905ebb1f)

Algolia URL seem to not be set correctly. This is what I get in the console
![image](https://github.com/serversideup/spin/assets/126659880/3db118a3-e08b-4089-8c20-2c5154f8072e)

**Comment by jaydrogers** [comment]
Thanks for pointing this out. We're tracking this issue here: https://github.com/serversideup/spin/issues/43
